docker/README.md

```diff
@@ -21,13 +21,13 @@ The versions of the main dependencies can be manipulated when building the Docke
 
 - `ARG DUNEVERSION=2.9`: DUNE version. Available in all recipes.
 - `ARG DUMUXVERSION=3.7`: DuMuX version. Available in all recipes.
-- `ARG PRECICEVERSION=develop`: preCICE version. Available only in "large" and "slim" recipes.
+- `ARG PRECICEVERSION=3.0.0`: preCICE version. Available only in "large" and "slim" (use branch name e.g. "v3.0.0" or "develop") recipes.
 - `ARG PRECICEUBUNTU=focal`: preCICE target platform (here, Ubuntu Focal Fossa). Available only in "large" recipe.
 
 If one wants to build a "slim" image using DUNE 2.9, DuMuX from the current master branch and preCICE 3.0.0 one could call it with the following command:
 
 ```text
-sudo docker build --build-arg DUNEVERSION=2.9 --build-arg DUMUXVERSION=master --build-arg PRECICEVERSION=3.0.0 -t precice/dumux-precice:master-3.0.0 --file dockerfile.slim .
+sudo docker build --build-arg DUNEVERSION=2.9 --build-arg DUMUXVERSION=master --build-arg PRECICEVERSION=v3.0.0 -t precice/dumux-precice:master-v3.0.0 --file dockerfile.slim .
```

Comment:
@IshaanDesai the workflow file would need versions such as "v3.0.0" which would change the tag pattern but the alternative is adding a "develop or release version" check as in `checkout_repositories.sh`. Any thoughts?

Reply:
Which workflow file are you referring to here?

Reply:
The two .slim dockerfiles use `PRECICEVERSION` as a branch/tag name (to allow for "develop"). In the .large dockerfile the variable is used to select a .deb package which doesn't include a "v".

Reply:
Ah okay. This is indeed inconsistent. Do we even use the `.large` files anywhere at all?

Reply:
I can only see the two .slim recipes being used.

Reply:
> I can only see the two .slim recipes being used.

In that case, I would suggest opening an issue about this inconsistency and finalizing the release for now.

Reply:
See #33.